lib/src/google_fonts_base.dart

```diff
@@ -161,15 +175,16 @@ GoogleFontsVariant _closestMatch(
 /// it is the first time it is being loaded.
 ///
 /// This function can return null if the font fails to load from the URL.
-Future<ByteData> _httpFetchFont(String fontName, String fontUrl) async {
+Future<ByteData> _httpFetchFontAndSaveToDevice(
+    String fontName, String fontUrl) async {
```

Comment:
nit: trailing comma and reformat.

```suggestion
    String fontName, String fontUrl,) async {
```